tests/conftest.py

```diff
@@ -436,6 +505,26 @@ def draft_order(order_with_lines):
     return order_with_lines
 
 
+@pytest.fixture
+def draft_order_no_shipping(order_with_lines_no_shipping):
+    order_with_lines_no_shipping.status = OrderStatus.DRAFT
+    order_with_lines_no_shipping.save(update_fields=['status'])
+    return order_with_lines_no_shipping
+
+
+@pytest.fixture
+def draft_order_weight_based(
+        draft_order, shipping_zone_weight_based, vatlayer):
+    method = shipping_zone_weight_based.shipping_methods.get()
+    draft_order.shipping_method_name = method.name
+    draft_order.shipping_method = method
+    draft_order.shipping_price = method.get_total(vatlayer)
+    draft_order.save(
+        update_fields=['shipping_method_name', 'shipping_method',
+                       'shipping_price'])
```

Comment:
`shipping_price` should be listed separately as `shipping_price_net` & `shipping_price_gross`